src/shared/bus-util.c

```diff
@@ -809,7 +809,17 @@ int bus_print_property(const char *name, sd_bus_message *property, bool value, b
 
                 if (strstr(name, "UMask") || strstr(name, "Mode"))
                         print_property(name, "%04o", u);
-                else
+                else if (streq(name, "UID")) {
+                        if (u == UID_INVALID)
```

Comment:
hmm uid_is_invalid() is the better checker for this I figure

Reply:
Hmmm, when I see `u == <constant>` then I know that the `<constant>` is the special value used as a placeholder. But `uid_is_valid()` is opaque, and feels more like a validation function. We should never allow uid to be equal to 0xFFFF, but if there was an error in our code, we'd want to report that as 0xFFFF, not [unset]. So... I can change it, if you think that's better, but it doesn't feel that way for me.